vizro-core/src/vizro/static/js/models/date_picker.js

```diff
@@ -7,3 +7,15 @@ export function _update_date_picker_values(value, input_store) {
   }
   return [value, value];
 }
+
+export function _update_date_picker_position(clicks) {
+  var element_id = window.dash_clientside.callback_context.inputs_list[0]["id"];
```

Comment:
How does this inputs_list look like? Like what else is stored there?

Reply:
inputs_list contains data about the callback inputs. 

<img width="368" alt="Screenshot 2024-03-14 at 15 18 46" src="https://github.com/mckinsey/vizro/assets/108531476/2052350b-7015-4239-9cb9-3696cdebf60a">